src/rdb/rdb_raft.c

```diff
@@ -1119,54 +1119,139 @@ rdb_raft_update_node(struct rdb *db, uint64_t index, raft_entry_t *entry, rdb_vo
 /* See rdb_raft_log_offer_single. */
 #define RDB_RAFT_ENTRY_NVOPS 2
 
+/* If entry == NULL, the caller doesn't want to apply the entry. */
```

Comment:
im not familiar with this code, but this comment makes it seem that one could call this function with entry == NULL?

Reply:
Oops, you're right: This comment was added for an earlier revision of this patch and is no longer applicable to the latest revision. I'll submit a doc-only patch to remove it. Thanks, @mchaarawi.

Reply:
https://github.com/daos-stack/daos/pull/14358